mani_skill/trajectory/replay_trajectory.py

```diff
@@ -399,6 +399,7 @@ def _main(
     # Load associated json
     json_path = traj_path.replace(".h5", ".json")
     json_data = io_utils.load_json(json_path)
+    env_kwargs["sim_config"]["scene_config"]["gravity"] = np.array(env_kwargs["sim_config"]["scene_config"]["gravity"])
```

Comment:
the correct fix is to add a scene config parser somewhere in the BaseEnv class. It seems

```
        self.sim_config = dacite.from_dict(data_class=SimConfig, data=merged_gpu_sim_config, config=dacite.Config(strict=True))

``` 
isn't doing the job? I'll let this pass for now and add an issue to fix this later